project-fob/Views/Home/MeetingPageHost.cshtml

```diff
@@ -1,192 +1,79 @@
-			}, 50);
-            
-		}
-
-		function buttonBackToNormal() {
-			var button = new google.visualization.PieChart(document.getElementById('button'));
-			button.draw(dataButton, optionsButton);
-		}
-
-		if (!isItDrawing) {
-			runUpdate();
-			isItDrawing = true;
-		}
-		var errorCounter = 0;
-		function runUpdate() { // host controller, refresh
-			setTimeout(runUpdate, 1000);
-			$.ajax({
-				type: "POST",
-				url: '@Url.Action("Refresh", "Host")',
+                url: '@Url.Action("Reset", "Host")',
+                data: { meetingIdString : "@ViewBag.meetingid" },
                 success: function (result) {
 
-                    updateChart(result, result);
-                    //alert("after if");
-                    var name = "Click for Next Topic, Fob count: " + result;
-						errorCounter = 0;
-						dataButton = google.visualization.arrayToDataTable([
-						['Language', 'Speakers (in millions)'],
-						[name, 100],
-						]);
-						button = new google.visualization.PieChart(document.getElementById('button'));
-						button.draw(dataButton, optionsButton);
-						google.visualization.events.addListener(button, 'select', clickHandler);
 				},
 				error: function () {
-					errorCounter++;
-					if (errorCounter >= 30) {
-						alert("Connection error. Returning to home page")
-						errorCounter = 0;
-						window.location = "index";
-					}
+					alert("Connection error");
 				}
 			});
 		}
+    
+        	var errorCounter = 0;
+    function runUpdate() { // host controller, refresh
+			        setTimeout(runUpdate, 1000);
+			        $.ajax({
+				        type: "POST",
+                        url: '@Url.Action("Refresh", "Host")',
+                        data: { meetingIdString : "@ViewBag.meetingid" },
+                        success: function (result) {
+                            //alert("hey");
+                            //This still need fixed
+                            //updateChart(result, result);
+
+                            var label = document.getElementById('fobcount');
+                            label.textContent = result;
+                            
+				        },
+                        error: function () {
+					        errorCounter++;
+					        if (errorCounter >= 30) {
+						        //alert("Connection error. Returning to home page")
+						        errorCounter = 0;
+						        window.location = "index";
+					        }
+				        }
+			        });
+    }
 
-		/*When it updates, for some reason it resizes the size of the pie chart in the background*/
-        function updateChart(total, leavers) {
-            if (leavers == 0) {
-                total = 1;
-            }
-			data = new google.visualization.arrayToDataTable([
-		   ['nothing', 'special here'],
-		   ['thisjust', parseInt(leavers)],
-		   ['thebackgroundchart', parseInt(total - leavers)]
-			]);
+    function hey() {
+        alert("hello there");
+        alert("General Kenobi");
+    }
```

Comment:
What's this `hey()` method about?

Reply:
https://www.youtube.com/watch?v=rEq1Z0bjdwc
I just left it there as an easter egg as originally I used it to check if the js working correctly in some situation